Create ingest-from-redhat-amq-streams.md (#2047)
risingwavelabs/risingwave-docs: docs/ingest/ingest-from-redhat-amq-streams.md

---
 id: ingest-from-redhat-amq-streams
 title: Ingest data from RedHat AMQ Streams
 description: Describes how to ingest data from RedHat AMQ Streams.
 slug: /ingest-from-redhat-amq-streams
---
<head>
  <link rel="canonical" href="https://docs.risingwave.com/docs/current/ingest-from-redhat-amq-streams/" />
</head>

You can ingest data from RedHat AMQ Streams into RisingWave by using the Kafka source connector in RisingWave.

 [AMQ Streams](https://developers.redhat.com/products/amq/overview) offers a distributed backbone that allows microservices and other applications to share data with extremely high throughput and extremely low latency, it is based on the  [Strimzi](http://strimzi.io/) and [Apache Kafka](http://kafka.apache.org/) projects. It has the features of publish and subscribe, long-term data retention, advanced queueing, replayable events, and, partition messages for scalability.

> ## Prerequisites[](https://docs.risingwave.com/docs/current/ingest-from-nats/#prerequisites)

Reviewer: Why did you add this link here?

Before ingesting data from RedHat AMQ Streams into RisingWave, please ensure the following:

- The AMQ Streams cluster is running and accessible from your RisingWave cluster.

> - Allow the AMQ Streams cluster so that RisingWave can ingest data If authentication is required for the cluster, make sure you have the client username and password.

Reviewer 2: Rewrite this sentence to be clearer.

> - Allow the AMQ Streams cluster so that RisingWave can ingest data. Ensure you possess the client username and password if authentication is necessary for the cluster.

Reviewer 2: I feel this item isn't very clear. The first sentence says the cluster is accessible from your RisingWave cluster, which is the item above. Therefore, I suggest using the same wording as the MQTT one.
```suggestion
-  If authentication is required for the AMQ Streams cluster, ensure you have the client username and password.
```
